index1.js

```diff
@@ -2,20 +2,22 @@
 var allButtons = document.querySelectorAll(".drum").length;
 
 //loop all the buttons under ".drum" class
-for (var i=0; i < allButtons; i++) {
-  //Select all the query that has been looped, then add an assign an event into a generic function
-  document.querySelectorAll(".drum")[i].addEventListener("click", function () {
-    //
+for (var i = 0; i < allButtons; i++) {
+  //Select all the query that has been looped, then add an event (click) into a generic function when the button being clicked
+  document.querySelectorAll(".drum")[i].addEventListener("click", function() {

```

Comment:
So if you used jquery, this is where you can just do $('.drum')

Reply:
ok i see now, so jquery is much effective